Marharyta_Miliuk/Module 3 - Java/Week 2/NewYearPresent/src/by/epam/cdp/java/nypresent/Runner.java

```diff
@@ -0,0 +1,66 @@
+package by.epam.cdp.java.nypresent;
+import java.util.InputMismatchException;
+
+import by.epam.cdp.java.nypresent.beans.Bar;
+import by.epam.cdp.java.nypresent.beans.ChocolateCandy;
+import by.epam.cdp.java.nypresent.beans.Lollypop;
+import by.epam.cdp.java.nypresent.utils.InputOutputStreams;
+
+public class Runner {
+	
+public static int askNumberOfCandies (){//creating method do define number of candies to put into present
+		
+		System.out.println("How many candies do you want?");
+		
+		return InputOutputStreams.scanNumbers();
+	}
+
+	public static void main (String []args){
+		
+	try{
+		
+		Present present = new Present ();
+		
+			int number = askNumberOfCandies();
+			
+			    for (int i = 0; i < number; i++){
+			    	
+				     InputOutputStreams.printMessage("Enter type of candy you want (Bar, Lollypop or Chocolate Candy)");//asking user to define object of which type to create
+				     String typeOfCandy = InputOutputStreams.scanMessage();
+				     
+				     InputOutputStreams.printMessage("Enter" + " " + (i+1)+ " " + "candy you want to add to your present");//asking user to define Name parameter for each created object
+				     String nameOfCandy = InputOutputStreams.scanMessage();
+				     
+				     switch(typeOfCandy){//using the switch/case to create objects of different types with the names user defined
+				    
+				           case "Bar":
+					       present.collection.add(new Bar(nameOfCandy));
```

Comment:
please think about more usability communication. If I enter "bar" but not "Bar" I'll get error.